### DISCUSSION TITLE
Grabbing/Following redirects?

### DISCUSSION THREAD

**Discussion by joaociocca** [epi052/feroxbuster#771] [General]
I was just noticing this today... Ferox won't follow redirects for URLs down the domain/path it's looking at?
I started testing it against a target, trying to come up with a few ideas, and noticed that when I access it on a browser or curl it, I get a redirection from https://domain.target to https://domain.target/further/down/path - but Ferox won't map/scan that.

**Comment by epi052** [comment]
there's some logic that prevents ferox from following a redirect that doesn't match the base scan's site. If the redirect is still on the same base scan's domain, and isn't following redirects, that might be a bug. 

Is there any variation on `domain.target`? like is it `sub.domain.target` or similar?

**Reply by joaociocca** [reply]
nope, exact same base. `domain.target` all the way. Anything I can do to troubleshoot and gather more info on my side?

**Reply by joaociocca** [reply]
OH DAMN. I forgot about `-r` 🤦‍♂️

**Reply by epi052** [reply]
Haha, it happens.  Things working with -r?

**Reply by joaociocca** [reply]
yeah... I gotta say "I don't know" now, because I advanced the script a bit more and made use of the `--stdin`. In a previous step I ran nmap and extracted URLs, including redirects, to a `urls` file, so that's over =p
